pkg/sql/run_control_test.go

```diff
@@ -850,6 +849,42 @@ func TestIdleInTransactionSessionTimeoutCommitWaitState(t *testing.T) {
 	}
 }
 
+func TestStatementTimeoutRetryableErrors(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	ctx := context.Background()
+
+	numNodes := 1
+	tc := serverutils.StartNewTestCluster(t, numNodes,
+		base.TestClusterArgs{
+			ReplicationMode: base.ReplicationManual,
+		})
+	defer tc.Stopper().Stop(ctx)
+
+	conn, err := tc.ServerConn(0).Conn(ctx)
+	require.NoError(t, err)
```

Comment:
looks like the `require` import was added in 39e7448ccddf92428716fbe56d257edaee02c874

i think you can just add the import here